scripts/mirror.js

```diff
@@ -527,9 +514,6 @@ const bumpVersion = (data, destination, source, originalData) => {
       case 'chrome_android':
         bumpFunction = bumpChromeAndroid;
         break;
-      case 'edge':
```

Comment:
This dispatch was kinda nice though. I think we can keep it by doing `bumpFunction = () => bumpEdge(extra, stuff, here)` and then avoid special-casing Edge below.

This is a bit it would be good to have @ddbeck check out, though.

Reply:
I'm happy to do this, but I'm thinking it would be better to follow this up in another PR, since it involves a bit more of a rewrite than I feel comfortable with in this PR.

Reply:
Hmm, I see, it's because the argument for `bumpEdge` isn't available here, right?

Reply:
I removed my earlier comment because I realized there actually __is__ a way to work it back in!  😅